.changeset/cold-seas-hunt.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vercel/blob": minor
+"vercel-storage-integration-test-suite": patch
+---
+
+Adds `abortSignal` option to all methods. This allows users to cancel requests using an AbortController and passing its signal to the operation.
```

Comment:
@luismeyer Can we also add usage in the release notes? It's always pleasant to have everything in the changelog directly. Something like:

```
Here's how to use it (`put` example):

```